src/Moryx.Runtime.Kernel/KernelServiceCollectionExtensions.cs

```diff
@@ -87,6 +93,19 @@ public static IConfigManager UseMoryxConfigurations(this IServiceProvider servic
             return configManager;
         }
 
+        /// <summary>
+        /// Use moryx file system and configure base directory
+        /// </summary>
+        /// <returns></returns>
+        public static IMoryxFileSystem UseMoryxFileSystem(this IServiceProvider serviceProvider, string path)
+        {
+            var fileSystem = serviceProvider.GetRequiredService<LocalFileSystem>();
```

Comment:
While the method states MORYX file system we actually use the LocalFileSystem. I don't know really what would be better here, just wanted to say it feels of 😅